components/blocks/FilesToCards.vue

```diff
@@ -243,9 +271,9 @@ export default {
     sortedArray() {
       let filtered = this.filteredData;
       if (this.searchGroup.length > 0) {
-        filtered = filtered.filter((card) => {
+        filtered = filtered.filter(() => {
```

Comment:
```suggestion
        filtered = filtered.filter((card) => {
```

Reply:
was there another comment near this one? I vaguely remember this suggestion being in tandem with a comment later on about things, but I can't find it.

Reply:
yes, this comment is irrelevant now. please ignore.